intern/remote/remote.py

```diff
@@ -14,6 +14,7 @@
 import six
 from abc import ABCMeta
 from six.moves import configparser
+from intern.service.mesh.service import MeshService
```

Comment:
we should convenience-import this so it's only `from intern.service import MeshService`. (I can show how to do this if helpful)